tests/test_solana_account.py

```diff
@@ -29,70 +28,52 @@ def mock_get_account_info(mocker: MockerFixture) -> AsyncMock:
     return async_mock
 
 
-def test_solana_account_update_with_rpc_response(
-    solana_pubkey: SolanaPublicKey, solana_client: SolanaClient
-) -> None:
-    actual = SolanaAccount(
-        key=solana_pubkey,
-        solana=solana_client,
-    )
-    assert actual.slot is None
-    assert actual.lamports is None
+def test_solana_account_update_with_rpc_response(solana_account: SolanaAccount) -> None:
+    assert solana_account.slot is None
+    assert solana_account.lamports is None
```

Comment:
```suggestion
```
Unless it is necessary for the logic, I think that testing the fixtures outside of the fixture code doesn't make much sense. wdyt?

Reply:
agree